docs/developer-docs/smart-contracts/maintain/settings.mdx

```diff
@@ -117,3 +117,107 @@ async fn modify_canister_settings() -> CallResult<()> {
 </TabItem>
 </AdornedTabs>
 
+## Controllers
+
+The `controllers` field in canister settings contains a list of principals that control the canister.
+Learn more about this field [here](docs/current/developer-docs/smart-contracts/maintain/control).
+
+## Compute allocation
+
+By default canisters are scheduled for execution in a "best-effort" manner.
+Canisters that require guaranteed execution can get a share of compute capacity by setting `compute_allocation` in their canister settings.
+Compute allocation is expressed in percents and denotes the percentage of an execution core reserved for the canister.
+For example, compute allocation of 50% means that the canister will get 50% of an execution core.
+It will be scheduled at least every other round.
+Compute allocation of 100% means that the canister will be scheduled every round.
```

Comment:
```suggestion
By default, canisters are scheduled for execution in a "best-effort" manner. Canisters that require guaranteed execution can get a share of compute capacity by setting `compute_allocation` in their canister settings.

Compute allocation is expressed in percents and denotes the percentage of an execution core reserved for the canister. For example, compute allocation of 50% means that the canister will get 50% of an execution core. It will be scheduled at least every other round. Compute allocation of 100% means that the canister will be scheduled every round.
```